extensions/oidc/runtime/src/main/java/io/quarkus/oidc/OidcTenantConfig.java

```diff
@@ -270,6 +270,14 @@ public Optional<String> getLeafCertificateName() {
         public void setLeafCertificateName(String leafCertificateName) {
             this.leafCertificateName = Optional.of(leafCertificateName);
         }
+
+        public Optional<String> getTrustStorePassword() {
+            return trustStorePassword;
+        }
+
+        public void setTrustStorePassword(String trustStorePassword) {
+            this.trustStorePassword = Optional.of(trustStorePassword);
```

Comment:
```suggestion
            this.trustStorePassword = Optional.ofNullable(trustStorePassword);
```

Reply:
@gastaldi Sure, thanks, I can go through all such cases in the config later as well. The pattern where an Optional is returned but is used internally in the setter may be a bit strange, but a long awhile back, the thinking it was not really a bean and it might save users a tiny bit of code, but indeed, if I were starting today, I'd just use Optional on the in and out :-)

Reply:
Ideally these classes could be converted into interfaces and we could avoid these setters